xcell/mappers/mapper_Planck_base.py

```diff
@@ -26,13 +26,23 @@ def _get_Planck_defaults(self, config):
         self.cl_coupled = None
         self.cls_cov = None
         self.custom_auto = True
+        self.pre_mask_ps = config.get("pre_mask_ps", False)
+        self.ps_mask = None
 
     def get_signal_map(self):
         if self.signal_map is None:
             signal_map = hp.read_map(self.file_map)
             signal_map[signal_map == hp.UNSEEN] = 0.0
             signal_map[np.isnan(signal_map)] = 0.0
             signal_map = rotate_map(signal_map, self.rot)
+            if self.pre_mask_ps is True:
+                if self.file_ps_mask is not None:
+                    ps_mask = self._get_ps_mask()
+                    signal_map *= ps_mask
+                else:
+                    NotImplementedError("""Tried to pre-mask point
+                                        sources but couldn't find
+                                        file_ps_mask""")
```

Comment:
All this should happen before `rotate_map`, since presumably the ps mask is in the original coordinates.